log-redaction.md

```diff
@@ -44,3 +44,7 @@ To enable log redaction in the TiKV side, set the value of [`security.redact-inf
 ## Log redaction in PD side
 
 To enable log redaction in the PD side, set the value of [`security.redact-info-log`](/pd-configuration-file.md#redact-info-log-new-in-v500-rc) to `true`. This configuration value defaults to `false`, which means that log redaction is disabled.
+
+## Log desensitization of TiFlash components
```

Comment:
Use the word "redaction" instead of "desensitization" to make it aligned with other components?